src/content/en/updates/2017/02/navigation-preload.md

```diff
@@ -86,7 +86,7 @@ service workers that don't have a fetch
 event](https://bugs.chromium.org/p/chromium/issues/detail?id=605844), by
 [launching service workers
 speculatively](https://codereview.chromium.org/2045153003), and other
-optimisations. However, bootup time will always be greater than zero.
+optimizations. However, bootup time will always be greater than zero.
```

Comment:
This is not necessarily a typo, but rather British spelling vs. American spelling. I’m personally fine with using American spelling consistently throughout the site, but I’ll defer to @jpmedley on such decisions.